torch/_dynamo/guards.py

```diff
@@ -1023,12 +1023,13 @@ def compile_check_fn(self, builder, guards_out, guard_fail_fn):
 
         # Don't report this guard, it's always the same, useless!
         code_parts = ["___guarded_code.valid", "___check_global_state()"]
+        verbose_code_parts = code_parts[:]
 
         def add_code_part(code, guard, log_only=False):
             extra = ""
             if guard.user_stack:
                 for fs in reversed(guard.user_stack):
-                    if fs.filename not in uninteresting_files():
+                    if fs.filename in uninteresting_files():
```

Comment:
Incorrectly skipped interesting files as opposed to uninteresting previously.

Reply:
Or is this supposed to be continue below?

Reply:
Meh, the old code is written too cleverly, but this version is not right either.

The intended semantics is: find the first interesting file, and set extra to format that frame.

The modified code you have here says "if the top of the user stack is an uninteresting file, immediately break. If every filename is interesting, set extra to the first frame."

This ends up not mattering in your test case because you only have one user frame. Try one with two user frames (inlined function call.)

Reply:
Right, I think we should not do else:, just assign extra when break.

Reply:
In particular, break means else statement is not executed.

```python
for i in range(10):
    if i == 5:
        break
else:
    print(10)  # not printed
```